packages/bot-web-ui/src/components/dashboard/dashboard.tsx

```diff
@@ -76,7 +76,7 @@ const Dashboard = observer(() => {
     };
     const active_hash_tab = GetHashedValue(active_tab);
 
-    const connectionDistrupted = () => {
+    const checkAndHandleConnection = () => {
         const api_status = api_base.getConnectionStatus();
         //added this check because after sleep mode all the store values refresh and is_running is false.
         const is_bot_running = document.getElementById('db-animation__stop-button') !== null;
```

Comment:
Don't we have a state to use to track if bot is running rather than checking the stop button if it exist? 
It's unlikely to happen but if ever later we change that button then this check won't work properly anymore.

Reply:
@prince-deriv I had already checked this for `is_running` from `run_panel_store` but for some reason the store values do not persist and puts it back to false and the condition never executes..